runners/core-java/src/main/java/org/apache/beam/runners/core/SideInputHandler.java

```diff
@@ -161,11 +162,6 @@ public void addSideInputValue(
   @Override
   public <T> T get(PCollectionView<T> sideInput, BoundedWindow window) {
 
-    if (!isReady(sideInput, window)) {
```

Comment:
Looks like @tgroh asked @aljoscha this same question in the [original PR](https://github.com/apache/beam/pull/2263#pullrequestreview-27893521)
@aljoscha ?